example.groovy

```diff
@@ -0,0 +1,5 @@
+class Example {
+    def sayHello() {
```

Comment:
📝 **[npm-groovy-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>def for method return type should not be used  NoDef